o8g/Scripts/actions.py

```diff
@@ -842,11 +842,20 @@ def updateBlessCurse():
     cb.markers[Bless] = b
 
 def addBlessCurse(group, isBless, who=me):
+    c = 0
+    b = 0
+
     mute()
     if chaosBag().controller != me:
         remoteCall(chaosBag().controller, "addBlessCurse", [group, isBless, me])
-        return
-
+        return  
+  
+    # Search Seal curse and bless token
+    for card in table:
+        if card.name == "Bless" and card.Subtype == "Sealed":
+            b += 1
+        if card.name == "Curse" and card.Subtype == "Sealed":    
+            c += 1     
```

Comment:
At first I thought this shouldn't be needed, but I now realize, that there are a few edge cases, where the number of sealed Blurse Tokens should also be counted. Still this should be part of updateBlessCurse() and the total added to the displayed value on the chaosbag.